web/src/lib/roadmap/buildUserRoadmap.test.ts

```diff
@@ -490,30 +490,46 @@ describe("buildUserRoadmap", () => {
     });
 
     describe("non essential questions", () => {
-      it("adds addOn if nonEssentialsQuestion value is true", async () => {
-        mockGetNonEssentiaQuestionAddOn.mockReturnValue("test-addOn-1");
+      it("adds addOn if nonEssentialQuestion value is true", async () => {
+        mockGetNonEssentialQuestionAddOn.mockReturnValue("non-essential-add-on-1");
         await buildUserRoadmap(
           generateStartingProfile({
-            industryId: "test-industry",
+            industryId: "non-essential-question-industry",
             nonEssentialRadioAnswers: {
-              "test-question-1": true,
+              "non-essential-question-1": true,
             },
           })
         );
-        expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).toContain("test-addOn-1");
+        expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).toContain("non-essential-add-on-1");
       });
 
-      it("doesn't add addOn if nonEssentialsQuestion is value is false", async () => {
-        mockGetNonEssentiaQuestionAddOn.mockReturnValue("test-addOn-2");
+      it("doesn't add addOn if nonEssentialQuestion value is false", async () => {
+        mockGetNonEssentialQuestionAddOn
+          .mockReturnValue("non-essential-add-on-2")
+          .mockReturnValue("non-essential-add-on-1");
         await buildUserRoadmap(
           generateStartingProfile({
             industryId: "test-industry",
             nonEssentialRadioAnswers: {
-              "test-question-2": false,
+              "non-essential-question-1": true,
+              "non-essential-question-2": false,
+            },
+          })
+        );
+        expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).not.toContain("non-essential-add-on-2");
+      });
+
+      it("doesn't add addOn if question id is not in industry nonEssentialQuestions", async () => {
+        mockGetNonEssentialQuestionAddOn.mockReturnValue("non-essential-add-on-3");
+        await buildUserRoadmap(
+          generateStartingProfile({
+            industryId: "non-essential-question-industry",
+            nonEssentialRadioAnswers: {
+              "non-essential-question-3": true,
```

Comment:
The test would be easier to read, imo, if you name this something descriptive rather than non-essential-question-3.

Reading this test on its own, it's not obvious why the expectation - why _shouldn't_ the addOns contain this one, when we're doing a test with "non-essential-question-industry" - which _sounds like_ it should have something like "non-essential-question-3" in it.

I have to scroll to the top of the file to understand that oh, that industry contains "non-essential-question-1" and "non-essential-question-2" but not "non-essential-question-3" which is a very subtle difference.

Anyways, all that to say, this would be much more obvious and require less investigation if you name it something like "question-not-contained-in-industry" or something. Names can be used to descriptively give intuition to the reader of what the purpose is.